src/hooks/useOrders.ts

```diff
@@ -0,0 +1,18 @@
+import { AuctionElement } from 'types'
+import { useWalletConnection } from './useWalletConnection'
+import useSafeState from './useSafeState'
+import { exchangeApi } from 'api'
+import { useEffect } from 'react'
+
+export function useOrders(): AuctionElement[] {
+  const { userAddress } = useWalletConnection()
+  const [orders, setOrders] = useSafeState<AuctionElement[]>([])
+
+  useEffect(() => {
+    userAddress && exchangeApi.getOrders(userAddress).then(setOrders)
+      setOrders(_orders)
```

Comment:
You just committed my suggestion ? :smile: Right here in github?

Multiline suggestions aren't here yet, but are on the github roadmap

Reply:
I did! Though it was ready to go, no?

Reply:
Ok now I see it. Fixed in latest revision :)